Matrix.SynapseInterop.Database/SynapseDbContext.cs

```diff
@@ -18,5 +31,52 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             optionsBuilder.UseNpgsql(_connString);
         }
+
+        public List<DeviceContentSet> GetNewDevicesForDestination(string destination, int limit)
```

Comment:
lint that doesn't exist: `IEnumberable` should be exposed, not a mutable collection